grunt/app_config.js

```diff
@@ -0,0 +1,14 @@
+module.exports = {
+  GruntConfigBuilder:
+    '/var/www/openorchestra/vendor/open-orchestra/open-orchestra-cms-bundle/GruntTasks/GruntConfigBuilder.js',
```

Comment:
For this one it could be relative, but tasksDir and targetDirs have to be absolute, so I put this one absolute too to be consistent